presto-benchmark/src/main/java/com/facebook/presto/benchmark/AbstractOperatorBenchmark.java

```diff
@@ -137,6 +144,8 @@ protected void execute(TaskContext taskContext)
 
         CpuTimer cpuTimer = new CpuTimer();
         execute(taskContext);
+        long peakMemory = this.peakMemory;
+        this.peakMemory = 0;
```

Comment:
Make `peakMemory` a local variable in `execute`. Make `execute` return a map: `ImmutableMap.of("peak_memory", peakMemory)`. Merge the map with the later constructed map below.